src/main/app/case/to-api-format.ts

```diff
@@ -4,12 +4,22 @@ import { Case, CaseDate, Checkbox, LanguagePreference, formFieldsToCaseMapping,
 import { CaseData, ConfidentialAddress, DivorceOrDissolution, Gender, YesOrNo } from './definition';
 import { theirAddressToApi, yourAddressToApi } from './formatter/address';
 
-type ToApiConverters = Partial<Record<keyof Case, string | ((data: Case) => Partial<CaseData>)>>;
+export type OrNull<T> = { [K in keyof T]: T[K] | null };
+
+type ToApiConverters = Partial<Record<keyof Case, string | ((data: Case) => OrNull<Partial<CaseData>>)>>;
+
+const checkboxConverter = (value: string | undefined) => {
+  if (!value) {
+    return null;
```

Comment:
would undefined not work?

Reply:
that way we might not need the or null stuff. tho i guess its best.

Reply:
I've updated this in 4860c11 to explicitly check for `null`. The only time this will be null is when the checkbox hasn't been ticked and the user clicks "Save and sign out".